runtime/doc/pattern.txt

```diff
@@ -71,14 +71,18 @@ N			Repeat the latest "/" or "?" [count] times in
 			Only whole keywords are searched for, like with the
 			command "/\<keyword\>".  |exclusive|
 			'ignorecase' is used, 'smartcase' is not.
+							*v_*-default*
```

Comment:
I didn't think about this before, but `v_*-default` won't work, since the help files use `*` to indicate tags. So it will actually need to be `v_star-default`.

Reply:
Thanks for your advice!